renderer-macos/lib/Brisk_macos.re

```diff
@@ -0,0 +1,21 @@
+module Brisk = Brisk;
+
+module Cocoa = {
+  module BriskApplication = BriskApplication;
```

Comment:
I think it makes sense to name the modules inside `Cocoa` *module* without the Brisk prefix.

Reply:
Hm, I agree

Reply:
Wait, then it will clash with corresponding components name.

Reply:
It won't - it's in the Cocoa module. I'd name the file `NSApplication.re`/`NSApplication.c` honestly.